sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1774,6 +1773,9 @@ def display_data(self):
       if self.table_reference.projectId is not None:
         tableSpec = '{}:{}'.format(self.table_reference.projectId, tableSpec)
       res['table'] = DisplayDataItem(tableSpec, label='Table')
+    method = self._compute_method(False) # TODO is there a way to get if its streaming at job launch time?
+    res['method'] = DisplayDataItem(
```

Comment:
perhaps you should just use `self.method` here, so that we can at least see DEFAULT, FILE_LOADS or STREAMING_INSERTS. Thoughts?